linux/net/rina/dtp-ps-default.c

```diff
@@ -312,11 +337,14 @@ dtp_ps_default_create(struct rina_component * component)
         ps->priv            = NULL;
 
         ps->transmission_control        = default_transmission_control;
+        ps->rx_control			= NULL;
         ps->closed_window               = default_closed_window;
-        ps->flow_control_overrun        = default_flow_control_overrun;
+        ps->snd_flow_control_overrun    = default_snd_flow_control_overrun;
+        ps->rcv_flow_control_overrun    = NULL;
```

Comment:
These policies were introduced by Miquel if I'm not wrong. 
We can discuss with him about this. The idea behind, if i'm not wrong, was to not block receiving data if the send rate have been fulfilled (and so maintain the mechanism separated).